test/uk/gov/hmrc/payetaxcalculatorfrontend/services/DetailedCalcEngineSpec.scala

```diff
@@ -106,6 +106,36 @@ class DetailedCalcEngineSpec extends FreeSpec with Matchers with PropertyChecks
         }
       }
     }
-  }
```

Comment:
Kolade already aproved the engine so this PR is not required. @kings13y do you see a value in keeping those tests?
There are 1 to 1 test with jira tickets requirements.